ivadomed/main.py

```diff
@@ -124,9 +124,9 @@ def get_dataset(bids_df, loader_params, data_lst, transform_params, cuda_availab
 def save_config_file(context, path_output):
     # Save config file within path_output and path_output/model_name
     # Done after the threshold_analysis to propate this info in the config files
-    with open(os.path.join(path_output, "config_file.json"), 'w') as fp:
+    with Path(path_output, "config_file.json").open(mode='w') as fp:
         json.dump(context, fp, indent=4)
-    with open(os.path.join(path_output, context["model_name"], context["model_name"] + ".json"), 'w') as fp:
+    with Path(path_output, context["model_name"], context["model_name"] + ".json").open(mode='w') as fp:
```

Comment:
Please ensure to adjust any future instances to use `.get()`  method to access dictionary. We really need to remove this antipattern.